### PR TITLE
omini-gtk-theme: init at unstable-2021-03-30

### PR DESCRIPTION
**Pull request by zoedsoupe**
###### Description of changes

Omni is a theme created for various applications. This package aims to build the tk version! https://github.com/getomni

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -25243,6 +25243,8 @@ with pkgs;
 
   oldsindhi = callPackage ../data/fonts/oldsindhi { };
 
+  omni-theme-gtk = callPackage ../data/themes/omni-theme-gtk { };
```
Comment (romildo) [inline pkgs/top-level/all-packages.nix:25246]:
```suggestion
  omni-gtk-theme = callPackage ../data/themes/omni-gtk-theme { };
```

**Review by romildo — CHANGES_REQUESTED** [review]
Some small changes.

File: pkgs/data/themes/omni-theme-gtk/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+let
+  themeName = "Omni";
+in
+stdenv.mkDerivation {
+  pname = "dracula-theme";
+  version = "unstable";
```
Comment (romildo) [inline pkgs/data/themes/omni-theme-gtk/default.nix:8]:
```suggestion
  pname = "omni-gtk-theme";
  version = "unstable-2021-03-30";
```

File: pkgs/data/themes/omni-theme-gtk/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  src = fetchFromGitHub {
+    owner = "getomni";
+    repo = "gtk";
+    rev = "HEAD";
```
Comment (romildo) [inline pkgs/data/themes/omni-theme-gtk/default.nix:13]:
```suggestion
    rev = "e81b3fbebebf53369cffe1fb662abc400edb04f7";
```

**Review by romildo — CHANGES_REQUESTED** [review]

**Comment by romildo** [comment]
Also rename the commit to:

omini-gtk-theme: init at unstable-2021-03-30

File: pkgs/data/themes/omni-gtk-theme/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+let
+  themeName = "Omni";
+in
+stdenv.mkDerivation {
```
Comment (SuperSandro2000) [inline pkgs/data/themes/omni-gtk-theme/default.nix:6]:
```suggestion
stdenv.mkDerivation {
```

File: pkgs/data/themes/omni-gtk-theme/default.nix [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  };
+
+  propagatedUserEnvPkgs = [
+    gtk-engine-murrine
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/themes/${themeName}
+    cp -a {assets,gnome-shell,gtk-2.0,gtk-3.0,gtk-3.20,index.theme,metacity-1,unity,xfwm4} $out/share/themes/${themeName}
```
Comment (SuperSandro2000) [inline pkgs/data/themes/omni-gtk-theme/default.nix:24]:
```suggestion

    mkdir -p $out/share/themes/Omni
    cp -a {assets,gnome-shell,gtk-2.0,gtk-3.0,gtk-3.20,index.theme,metacity-1,unity,xfwm4} $out/share/themes/Omni
```

**Comment by romildo** [comment]
Result of `nixpkgs-review pr 184020` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>omni-gtk-theme</li>
  </ul>
</details>

**Review by romildo — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg build omni-gtk-theme

**Pull request merged**